package.json

```diff
@@ -156,14 +156,15 @@
     "piping": "0.3.2",
     "react": "15.3.1",
     "react-addons-css-transition-group": "15.3.1",
+    "redux-async-connect": "1.0.0-rc4",
```

Comment:
Whoops, no, it was from me tweaking some `npm` version oddness. I'll remove 'em.